css/src/atomics/display.scss

```diff
@@ -0,0 +1,27 @@
+@each $item in $displays {
+	.display-#{$item} {
+		display: #{$item} !important;
+	}
+
+	.display-#{$item}-tablet {
+		@include tablet {
+			display: #{$item} !important;
+		}
+	}
+
+	.display-#{$item}-desktop {
+		@include desktop {
+			display: #{$item} !important;
+		}
+	}
+
+	.display-#{$item}-widescreen {
+		@include widescreen {
+			display: #{$item} !important;
+		}
+	}
+}
+
+[hidden] {
```

Comment:
Are there plans to also add the `is-hidden` is-invisible` classes?

Reply:
I could see `.display-none` being added to match with our atomic class structure.

Reply:
Added 'none' as well